Polyhedron/demo/Polyhedron/Plugins/PCA/Scene_edit_box_item.cpp

```diff
@@ -1024,12 +1024,12 @@ void Scene_edit_box_item_priv::reset_selection()
 //intercept events for picking
 bool Scene_edit_box_item::eventFilter(QObject *, QEvent *event)
 {
+  QGLViewer* viewer = *QGLViewer::QGLViewerPool().begin();
```

Comment:
`viewer` should be equal to the first argument `QObject *` of the eventFilter. The more we use `QGLViewerPool()`, the more difficult it will be to have multiple viewers.